### ISSUE TITLE
Board Game API

### ISSUE THREAD

**Issue by bartelto** [bartelto/project-2#11]
- [ ] Choose an API.
- [ ] Will we transfer board game data to our DB?
- [ ] What board game properties will we use?

**Comment by ashleyerffmeyer** [comment]
Board Game Atlas API works - currently displays board game images! I assume we want to show board games images for only the games profiles match on, but would like to confirm this. If we want to show board game images for all board games on each profile, using a database is probably more idea.

**Comment by bartelto** [comment]
We can discuss it with the others, but I don't think we should include game board images in the MVP. My reason: the focus is on finding other players, not exploring new games.

**Comment by ashleyerffmeyer** [comment]
Yeah, would be good to discuss. Since the functionality is already there, it would be easy to plug in whatever game(s) 2 users match on and search for that image with the API. Either way, it sounds like the next thing we need to work on is the matching functionality. I don't mind starting on that since in my mind it's similar to friend finder. My friend finder works now up to the point of displaying the modal. However, the modal displays my info rather than the friend's at the moment. Can ask for help when I get to that part! lol.
